reana_client/utils.py

```diff
@@ -38,7 +40,7 @@ def workflow_uuid_or_name(ctx, param, value):
         return value
 
 
-def yadage_load(workflow_file, toplevel='.'):
+def yadage_load(workflow_file, toplevel='.', **kwargs):
```

Comment:
Who is consuming these `**kwargs`? We already unpack `toplevel` which is the only one we are interested in.

Reply:
If we set other `inputs.options` such as `initdir`, [its value will get here](https://github.com/reanahub/reana-client/pull/400/files#diff-670175eb24e7fca435fba316a6b7066fR137) and this function would fail. With `**kwargs` we can _filter out_ these options that are not needed on validation time. I agree it's a bit hacky but it's cleanest and safest way I could come up with. If you have any other idea please let me know 🙂